usr/src/uts/common/fs/zfs/sys/zfs_ioctl.h

```diff
@@ -372,7 +372,9 @@ typedef struct zfs_cmd {
 	uint64_t	zc_action_handle;
 	int		zc_cleanup_fd;
 	uint8_t		zc_simple;
+	uint8_t		zc_pad3[3];
 	boolean_t	zc_resumable;
+	uint32_t	zc_pad4;
```

Comment:
That doesn't work; zc_pad2 is needed to ensure that zc_history is 64-bit aligned when compiled 32-bit.